### ISSUE TITLE
"Back" option not obvious when viewing a Project

### ISSUE THREAD

**Issue by joepavitt** [flowforge/flowforge#646]
### Current Behavior

Currently, the "Project" page looks like this:

<img width="1135" alt="Screenshot 2022-06-01 at 11 34 25" src="https://user-images.githubusercontent.com/99246719/171385402-957db22f-ff78-4624-83d5-15443a892e96.png">

In order to go back to the "Team" view, you need to click the team name. This is inconsistent with all other pages where we display a "Back to ..." button.

### Expected Behavior

We should add a "Back to Dashboard" option between the team selection box and the project name in the side menu.

### Environment

- FlowForge version: 0.5.0
- Node.js version:
- npm version:
- Platform/OS:
- Browser:

**Comment by knolleary** [comment]
I agree the sidebar needs improving on Project pages, but I'm not sure a back button is the right answer. We already covered this in issue https://github.com/flowforge/flowforge/issues/533 based on the previous iteration of the page.

The specific feedback was:

> The 'Back to Projects' link at the top assumes you've arrived at this page from the Team Projects page. That isn't necessarily true as you can reach a Project via various links. Having a back button take me somewhere different to where I came from doesn't feel right

My main feedback with the current side bar is the project name is tiny compared to all the other elements. It should be more prominent and obvious to give a better sense of location and hierarchy.

I was going to propose something along these lines - that when on a Project page, the "Team" entry should be smaller (not the full team switch widget), and the Project entry should be the most prominent. I can mock something up to better communicate what I mean.

**Comment by sammachin** [comment]
This issue needs considering in conjunction with #690 and #616 as they all relate to navigation around the team context.
We should look for a change that satisfies all of them if possible.
@joepavitt happy to have a live discussion around this if it would help?

**Comment by knolleary** [comment]
This is 'done' state - but not sure what PRs resolved it or if this can now be closed?
